serverless/installing-service-mesh.adoc

```diff
@@ -0,0 +1,17 @@
+// Module included in the following assemblies:
+//
+// serverless/installing-openshift-serverless.adoc
+
+[id=installing-service-mesh-operator_{context}]
+= Installing the ServiceMesh operator
+
+Openshift Serverless requires an installed version of ServiceMesh.
+
+include:modules/ossm-operator-install.adoc[leveloffset=+1]
+
+====
+[NOTE] 
+ServiceMesh documentation requests the installation of Kiali and Jaeger Operators. Kiali and Jaeger operators do not need to be installed unless you plan to use them. They are disabled in the default settings proposed below.
+====
```

Comment:
Should be in this form:

```
[NOTE]
====
ServiceMesh documentation requests the installation of Kiali and Jaeger Operators. Kiali and Jaeger operators do not need to be installed unless you plan to use them. They are disabled in the default settings proposed below.
====
```

Note that there also appears to be a whitespace after `[NOTE] ` above; should be removed.

Reply:
There's a few other instances of this out-of-order `[NOTE`] and `====` blocks.